### PR TITLE
Vendicated.Vencord version 1.2.9

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107243)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
> 2023-05-18 02:44:05.925 [CLI ] ShellExecute installer failed: 1 
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230518094405

**Comment by solomoncyj** [comment]
ok

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by solomoncyj** [comment]
.

**Comment by stephengillie** [comment]
> Automatic Validation ended with:
> 
> > Installation failed with exit code -1978335226
> > 2023-05-18 02:44:05.925 [CLI ] ShellExecute installer failed: 1
> > InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230518094405

This is unchanged.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2023-08-28 17:18:51.076 [CORE] Download completed.  
> 2023-08-28 17:18:51.077 [CLI ] Starting installer selection.  
> 2023-08-28 17:18:51.077 [CLI ] Installer [X86,exe,User,en-US] not applicable: Architecture was excluded by caller : X86  
> 2023-08-28 17:18:51.077 [CLI ] No suitable installer found for manifest Discord.Discord with version 1.0.9016  
> 2023-08-28 17:18:51.078 [CLI ] Terminating context: 0x8a150104 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DependenciesFlow.cpp:12b

File: manifests/v/Vencord/Vencord/1.2.10/Vencord.Vencord.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Vencord.Vencord
+PackageVersion: 1.2.10
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Discord.Discord
+Installers:
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/v/Vencord/Vencord/1.2.10/Vencord.Vencord.installer.yaml:10]:
```suggestion
- Architecture: x86
```

Comment (stephengillie) [reply manifests/v/Vencord/Vencord/1.2.10/Vencord.Vencord.installer.yaml:10]: Discord.Discord only has an x86 version.

**Comment by solomoncyj** [comment]
@stephengillie

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
2023-08-30 06:45:00.118 [CLI ] ShellExecute installer failed: 1 
image

**Comment by stephengillie** [comment]
This installed normally in a Windows 10 VM. I think the error might be due to the dependency.

**Comment by solomoncyj** [comment]
Can someone pls waiver this? Thanks

**Review by stephengillie — APPROVED** [review]

**Pull request merged**